es-ES/readme.md

```diff
@@ -0,0 +1,703 @@
+
+[![Build Status: Linux](https://travis-ci.org/sindresorhus/ava.svg?branch=master)](https://travis-ci.org/sindresorhus/ava) [![Build status: Windows](https://ci.appveyor.com/api/projects/status/igogxrcmhhm085co/branch/master?svg=true)](https://ci.appveyor.com/project/sindresorhus/ava/branch/master) [![Coverage Status](https://coveralls.io/repos/sindresorhus/ava/badge.svg?branch=master&service=github)](https://coveralls.io/github/sindresorhus/ava?branch=master) [![Gitter](https://img.shields.io/badge/Gitter-Join_the_AVA_chat_%E2%86%92-00d06f.svg)](https://gitter.im/sindresorhus/ava)
+
+A pesar de que JavaScript se ejecuta en un solo hilo, IO en Node.js puede ejecutarse en paralelo debido a su naturaleza asíncrona. AVA se aprovecha de esto y corre sus tests al mismo tiempo, lo que es especialmente beneficioso para tests pesados en IO. Además, los archivos de test se ejecutan en paralelo como procesos separados, que le da un mejor rendimiento y un entorno aislado para cada archivo de test. [Cambiando](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) de Mocha a AVA en Pageres llevó el tiempo de test por debajo de 31 segundos, concretamente a 11 segundos. El tener tests que se ejecutan al mismo tiempo nos obliga a escribir tests atómicos, es decir, los tests no dependen del estado global o el estado de otros tests, lo que está muy bien!
+
+*Lea nuestra [guía de contribución](contributing.md) si está pensando en contribuir (issues/PRs/etc).*
+
+
+## Tabla de contenidos
+
+- [Uso](#usage)
+- [Uso de CLI](#cli)
+- [Documentación](#documentation)
+- [API](#api)
+- [Aserciones](#assertions)
+- [FAQ](#faq)
+
+
+## ¿Por qué AVA?
+
+- Mínimo y rápido
+- Sintaxis simple de test
+- Ejecuta tests concurrentemente
+- Fuerza a escribir tests atómicos
+- No globales implícitos
+- [Entorno aislado para cada archivo de test](#isolated-environment)
+- [Escribe tus tests en ES2015](#es2015-support)
+- [Soporte a Promesas](#promise-support)
+- [Soporte a Generadores](#generator-function-support)
+- [Soporte a Async](#async-function-support)
+- [Soporte a Observable](#observable-support)
+- [Aserciones mejoradas](#enhanced-asserts)
+- [Salida TAP opcional](#optional-tap-output)
+
+
+## Sintaxis de test
+
+```js
+import test from 'ava';
+
+test(t => {
+	t.same([1, 2], [1, 2]);
+});
+```
+
+
+## Uso
+
+#### Inicializar
+
+Instalar AVA globalmente `$ npm install --global ava` y ejecute `$ ava --init` (con cualquiera de las opciones) para añadir AVA a tu package.json o cree una.
+
+```json
+{
+	"name": "awesome-package",
+	"scripts": {
+		"test": "ava"
+	},
+	"devDependencies": {
+		"ava": "^0.6.0"
+	}
+}
+```
+
+#### Crear tu archivo de test
+
+```js
+import test from 'ava';
+import delay from 'delay';
+
+test('foo', t => {
+	t.pass();
+});
+
+test('bar', async t => {
+	t.plan(1);
+
+	const bar = Promise.resolve('bar').then(delay(200));
+
+	t.is(await bar, 'bar');
+});
+```
+
+<img src="screenshot.png" width="150" align="right">
```

Comment:
To see correctly the image :
`<img src="screenshot.png" width="150" align="right">`
replace by : 
`<img src="https://github.com/sindresorhus/ava/blob/master/screenshot.png" width="150" align="right">`